### ISSUE TITLE
Comparator on quantum bridge

### ISSUE THREAD

**Issue by Cobra1117** [AppliedEnergistics/Applied-Energistics-2#730]
It would be nice if you could use a redstone comparator to determine if a quantum bridge is active. (Or some other method of emitting a redstone signal if the bridge is online/offline.) 

Here's a common use case:

Main base: Has lots of power generation and the main AE system
Quarry: Has a small power generation system, a quarry, and a quantum bridge back to the base

Obviously I don't want to use my quarry's power generation to power the AE system, but I do need to provide the bridge power to get it connected. The problem occurs when the server is restarted (or when your AE system temporarily goes down)--some of the quantum bridges reset and you have to manually go around powering them back up. However, if you could monitor the status of the bridge directly this could be avoided. 

The only ways that I've come up with to do this with the existing system are to either keep X of an item (say torches) in the main AE system and use level emitters with large backup batteries. Or have a redstone P2P with a clock that resets a timer on a regular basis. (Both of which are somewhat involved for such a common problem.)

NOTE: You can't fix this with simply having an "always on" redstone P2P from the main base to the quarry because the redstone P2P freezes in its current state when it looses its P2P connection.

**Comment by Cisien** [comment]
Once a bridge is powered external it will be able to draw power from the other side of the link. All you need to do to keep the bridge from resetting is to place an energy cell (possibly dense) on the distant end of the link

**Comment by Cobra1117** [comment]
I had problems with a setup like that, but it might be a case of bad chunk loading. 

It still would be a nice feature to have, but if it's too hard and/or not worthwhile to folks, that's fine. :-)

**Comment by Zaflis** [comment]
If you chunkload the whole AE network on both sides of the bridges, it should smoothly come online on server start. If it doesn't, you have a problem with the setup or experiencing a bug.

**Comment by yueh** [comment]
A comparator has always the meaning of something is inside the inventory, which is true for the QNB itself. The strength should probably be 15 and not 1 as it can only hold a single item, but I am not sure if Minecraft by default uses the general stack limit of an item or the one defined by the inventory.

You can also argue that most energy storage are supporting comparators, but a QNB has no storage itself to monitor. (Making me aware that our cells are not supporting it.)

And starting to encode different states as strength makes it quite unintuitive. Like is a strength of 1 now has power but no singularity or the other way? 2 could be has power + has singularity + not linked or has singularity + is linked but no power?
Basically these states need to be uniquely encoded:
* is formed?
* is powered?
* singularity?
* is linked?

Which results in 16 different states. Just enough to fit into vanilla redstone, if I did not forget something. Some of them are even non valid states, like anything not formed. If this is the LSB, it would mean every even strength is an invalid state. If MSB, every strength > 0 or < 8 is invalid.

This feature should really be related to level emitters. But level emitters are currently not switching off, if the network is running out of power.

**Comment by Cobra1117** [comment]
@yueh That's a good point--I could solve any problems on my side by having comparator support on the energy cells. (If the the remote base has low power in its cell, I can route power into the local network until the cell fills back up.)

**Comment by yueh** [comment]
Related: #733
